salt/_modules/kubernetes.py

```diff
@@ -1975,3 +1992,162 @@ def replace_daemonset(
             raise CommandExecutionError(exc)
     finally:
         _cleanup(**cfg)
+
+
+def show_clusterrole(name, **kwargs):
+    cfg = _setup_conn(**kwargs)
+    try:
+        api_instance = kubernetes.client.RbacAuthorizationV1Api()
+        api_response = api_instance.read_cluster_role(name)
+
+        return api_response.to_dict()
```

Comment:
If there is not evaluation/remote call in the `to_dict` call, it's probably better to put it into an `else` statement.

Reply:
Yes we just copy/pasted from salt module for the moment. But yes indeed